tests/integration/conftest.py

```diff
@@ -65,18 +65,22 @@ def split_sample_data_dir(tmpdir_factory):
         # We are special casing the SSA dataset because that is the only one without the
         # household_id columns
         outdir = split_sample_data_dir.mkdir(dataset_name)
-        if dataset_name == DatasetNames.SSA:
+        if dataset_name in [
```

Comment:
These changes don't seem related to the SSA observer column changes you made. Are they a merge issue?

Reply:
This was the issue I raised on Friday - there was no sorting when splitting on household ids so most of the mhousehold ids that split the new sample data were in one of the shards we split the data in. This resulted in not having enough data fin the second shard to make a 1040 dataset in some integration tests because we then would filter down to a specific year.